config/initializers/assets.rb

```diff
@@ -7,7 +7,7 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-Rails.application.config.assets.paths << Rails.root.join('node_modules/bootstrap-icons/font')
+Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font")
```

Comment:
I would probably keep using `'` to have it consistent with the rest of the project. But that's a minor thing, feel free to just ignore this comment.

Reply:
I prefer as well `"` , but in this case one of compiler automatically adding new line with `"`